modules/customize-certificates-add-service-serving.adoc

```diff
@@ -5,15 +5,16 @@
 [id="add-service-certificate_{context}"]
 = Add a service certificate
 
-To secure communication to your service, generate a
-signed serving certificate and key pair into a secret in the same
-namespace as the service.
+To secure communication to your service, generate a signed serving certificate and key pair into a secret in the same namespace as the service.
+
+The generated certificate is only valid for the internal service DNS name `<service.name>.<service.namespace>.svc`, and is only valid for internal communications. If your service is a headless service (no `clusterIP` value set), the generated certificate also contains a wildcard subject in the format of `*.<service.name>.<service.namespace>.svc`.
```

Comment:
Sort of out of scope since this is just moved, mostly: should the user replaceable values have italics, and `_` instead of `.`?

Reply:
Not italics since we haven't adopted that yet for replaceable values in OpenShift docs. 

Good question about the underscores instead of periods - I actually had considered updating that, but I think it's really saying the `name` and `namespace` fields of the service object, so I left it alone.